doc/api/next_api_changes/deprecations/25427-AL.rst

```diff
@@ -0,0 +1,3 @@
+``GridHelperCurveLinear.get_data_boundary``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+... is deprecated with no replacement.
```

Comment:
Do we want a note about how one can do this still?

Use `grid_finder.extreme_finder(*[None] * 5)` to get the extremes of the grid.

Reply:
sure, done.